coreUtilities/src/testFixtures/java/org/opensearch/migrations/logging/CloseableLogSetup.java

```diff
@@ -0,0 +1,52 @@
+package org.opensearch.migrations.logging;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+import org.apache.logging.log4j.Level;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.core.LogEvent;
+import org.apache.logging.log4j.core.appender.AbstractAppender;
+import org.slf4j.LoggerFactory;
+
+import lombok.Getter;
+
+public class CloseableLogSetup implements AutoCloseable {
```

Comment:
Since this is getting published as part of the test fixtures could we add java docs?

Reply:
Done to both